src/main/java/org/fcrepo/migration/Migrator.java

```diff
@@ -37,27 +38,56 @@ public static void main(String [] args) throws IOException, XMLStreamException {
             Migrator m = new Migrator(new FoxmlDirectoryObjectSource(foxmlRoot, dsRoot, working),
                     new ConsoleLoggingStreamingFedoraObjectHandler());
             m.run();
-        }
+        }*/
+        final ApplicationContext context = new ClassPathXmlApplicationContext("spring/migration-bean.xml");
+        final Migrator m = (Migrator)context.getBean("migrator");
+        System.out.println("After creating bean...");
+        m.run();
+        ((ConfigurableApplicationContext)context).close();
```

Comment:
`ClassPathXmlApplicationContext` is a `ConfigurableApplicationContext`. Why do you upcast it to a variable of type `ApplicationContext` and then downcast back to `ConfigurableApplicationContext`?